config_bucket.go

```diff
@@ -31,13 +31,18 @@ func (b BucketType) validate() error {
 type BucketsConfig struct {
 	LogBucket BucketConfig `bson:"log_bucket" json:"log_bucket" yaml:"log_bucket"`
 
+	// InternalBuckets are the buckets that Evergreen's app servers have access to
+	// via their IRSA role.
+	InternalBuckets []string `yaml:"internal_buckets" bson:"internal_buckets" json:"internal_buckets"`
```

Comment:
The way `BucketsConfig` is structured now this seems out of place. Like before the `Credentials` field contains the credentials for accessing the `LogBucket`. But now the `InternalBuckets` are interjected in the middle.

Reply:
Thanks for the catch! I swapped it back and added explicit comments so future goers don't do the same mistake (as easily at least)